### PR TITLE
Script: fixed exception handling within isSentToMultiSig() method

### PR DESCRIPTION
**Pull request by natzei**
It fixes issue #1328

### CODE REVIEW

**Comment by schildbach** [comment]
Do you know where this IllegalArgumentException is thrown? Maybe we should mark this in a code comment or narrow the try block down to the line?

**Comment by alecalve** [comment]
@schildbach A partial stacktrace is available at #1328

**Comment by natzei** [comment]
`decodeFromOpN` throws the exception: the method uses `Preconditions.checkArgument`.

File: core/src/main/java/org/bitcoinj/script/Script.java [hunk 1]
```diff
@@ -717,7 +717,7 @@ public boolean isSentToMultiSig() {
             }
             // First chunk must be an OP_N opcode too.
             if (decodeFromOpN(chunks.get(0).opcode) < 1) return false;
-        } catch (IllegalStateException e) {
+        } catch (IllegalArgumentException e) {
             return false;   // Not an OP_N opcode.
```
Comment (natzei) [inline core/src/main/java/org/bitcoinj/script/Script.java]: If you consider this comment, the intended behaviour was to catch that exception.

**Comment by schildbach** [comment]
Thanks a lot, merged!

**Pull request closed without merging**